src/main/java/com/backend/soullive_a/entity/ModelImageKeyword.java

```diff
@@ -0,0 +1,29 @@
+package com.backend.soullive_a.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+
+public class ModelImageKeyword {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long modelImageKeywordId;
```

Comment:
```suggestion
    @Column(name = "MODEL_IMAGE_KEYWORD_ID")
    private Long id;
```
기본 아이디는 id라고 써주셔도 될 것 같아요 다른클래스도 다..!

Reply:
네! erd에 저렇게 되어있길래 저렇게 쓰는 줄 알았어요